test_fractions.py

```diff
@@ -796,11 +797,50 @@ def test_pi_digits(self):
             self.assertEqual(ff.numerator, qf.numerator)
             self.assertEqual(ff.denominator, qf.denominator)
 
+class CImportTest(unittest.TestCase):
+
+    def setUp(self):
+        self.build_test_module()
+
+    def tearDown(self):
+        self.remove_test_module()
+
+    def build_test_module(self):
+        self.module_code = '\n'.join([
```

Comment:
Why is this an instance attribute instead of a local variable? (Same for `pyx_importer` etc.)

Reply:
There's no reason it can't be local.  I'll change it.
Please excuse my lack of unittest writing, I mostly work with tests driven solely off of pytest.